src/libs/OptionsListUtils.js

```diff
@@ -230,10 +230,11 @@ function createOption(personalDetailList, report, draftComments, {
             ? lastMessageText
             : Str.removeSMSDomain(personalDetail.login);
     }
+    const icons = (report && report.icons && report.icons[0]) ? report.icons : [personalDetail.avatar];
```

Comment:
Can you please use `Lodash.get` instead for this? I think it would make this more simple.

Reply:
Sure!